sandy/sections/mf2.py

```diff
@@ -167,11 +167,15 @@ def read_mf2(tape, mat):
             info = {}
             C, i = sandy.read_cont(df, i)
             header2 = {
-                    "LRU": C.L1,            #Flag indicating whether this energy range contains data for resolved or unresolved resonance parameters:
-                    "LRF": C.L2,            #Flag indicating which representation has been used for the energy range. 
-                    "NRO": C.N1,            #Flag designating possible energy dependence of the scattering radiu
-                    "NAPS":C.N2,            #Flag controlling the use of the two radii
-                    }
+                # Flag indicating whether this energy range contains data for
+                # resolved or unresolved resonance parameters:
+                "LRU": C.L1,
```

Comment:
Item L1 of a cont record like LRU should already be a an integer, why do you do `LRU = int(C.L1)` below